en/02_Developer_Guides/00_Model/08_SQL_Select.md

```diff
@@ -105,6 +105,9 @@ $sqlQuery->selectField('YEAR("Birthday")', 'Birthyear');
 $joinOnClause = $schema->sqlColumnForField(Player::class, 'TeamID') . ' = ' . $schema->sqlColumnForField(Team::class, 'ID');
 $sqlQuery->addLeftJoin($teamTableName, $joinOnClause);
 
+// Combine another query using a union
+$sqlQuery->addUnion($anotherSqlSelect, SQLSelect::UNION_ALL);
```

Comment:
I've intentionally not added any more documentation than this for unions because I think the method is self-explanatory for anyone who knows what a union is and how it's used in SQL.

Frankly, anyone who doesn't understand the SQL concept of unions probably won't care about it anyway or have any reason to use it.

This is also roughly the same amount of documentation available for joins, which are arguably more complex to use as far as the API is concerned.